src/layers/VivViewerLayer/VivViewerLayer.js

```diff
@@ -50,7 +50,13 @@ export default class VivViewerLayer extends CompositeLayer {
       const tile = await loader.getTile({
         x,
         y,
-        z: -z,
+        // I don't fully undertstand why this works, but I have a sense.
+        // It's basically to cancel out:
+        // https://github.com/visgl/deck.gl/pull/4616/files#diff-4d6a2e500c0e79e12e562c4f1217dc80R128,
+        // which felt odd to me to beign with.
+        // The image-tile example works without, this but I have a feeling there is something
+        // going on with our pyramids and/or rendering that is different.
+        z: -z + Math.log2(512 / tileSize),
```

Comment:
hmmm I'm a bit confused here. Does this work for all sized tiles?

Reply:
Do they need to be powers of two?

Reply:
I honestly haven't tried but if you look in the deck.gl link I posted, you'll see this same transformation.  What the comment says is that I don't know why it has to be done again, but to get things to work, it does.

Reply:
I think you might be on to something in that I need to add a `round` operation.  Thanks!